src/main/java/com/easypost/service/EasyPostClient.java

```diff
@@ -125,8 +166,17 @@ public String getApiKey() {
     }
 
     /**
-     * Get API base for this EasyPostClient object.
+     * Get API version for this EasyPostClient object.
      * 
+     * @return the API version for this EasyPostClient object.
+     */
+    public String getApiVersion() {
```

Comment:
We don't need getters for the API version. Make the `apiVersion` property public and final, so it can't be overridden by end-users (only set via the constructor)

Reply:
Alternatively, if we are going to keep this getter, let's have it accept a parameter so it can return `v2` by default, but other possible versions (i.e. `beta`) on request.

Reply:
We should not have an additional parameter in the getter method, this will allow end users to change the apiVersion variable. The reason why the `apiVersion` is not final so we could change it internally in the `EasyPostClient` if the namespace has `beta`, e.g: `client.beta.endshipper.create()`

Reply:
I foresee some issues arising when we do need to implement a new beta service down the road with this current design, but we'll kick the can on that for now.